### PR TITLE
Update code with latest changes from Marvell LSP 18.12

### PR DESCRIPTION
**Pull request by kostapr**
Sync with Marvell branch "atf-v2.0-devel-18.12"

### CODE REVIEW

File: plat/marvell/a8k/common/a8k_common.mk [hunk 1]
```diff
@@ -13,6 +13,9 @@ PLAT_COMMON_BASE	:= $(PLAT_FAMILY_BASE)/common
 MARVELL_DRV_BASE	:= drivers/marvell
 MARVELL_COMMON_BASE	:= plat/marvell/common
 
+SVC_TEST		:= 0
```
Comment (antonio-nino-diaz-arm) [inline plat/marvell/a8k/common/a8k_common.mk]: You should prefix this option by MARVELL.

**Comment by antonio-nino-diaz-arm** [comment]
IP Review +1

File: plat/marvell/a8k/common/a8k_common.mk (hunk 1 above)
Comment (kostapr) [reply plat/marvell/a8k/common/a8k_common.mk]: Done

File: include/drivers/marvell/ap807_clocks_init.h [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (C) 2018 Marvell International Ltd.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ * https://spdx.org/licenses
+ */
+
+#ifndef AP807_INIT_CLOCKS_H_
```
Comment (antonio-nino-diaz-arm) [inline include/drivers/marvell/ap807_clocks_init.h]: ``AP807_CLOCKS_INIT_H``

File: drivers/marvell/comphy/phy-comphy-common.h [hunk 3]
```diff
@@ -23,8 +23,10 @@
  *      - bit 11~8 represent unit index
  *      - bit 16~12 represent mode
  *      - bit 17 represent comphy indication of clock source
- *      - bit 19-18 represents pcie width (in case of pcie comphy config.)
- *      - bit 31~20 reserved
+ *      - bit 20~18 represents pcie width (in case of pcie comphy config.)
+ *	- bit 21 represents the source of the request (Linux/Bootloader),
```
Comment (antonio-nino-diaz-arm) [inline drivers/marvell/comphy/phy-comphy-common.h]: Align this.

Comment (kostapr) [reply drivers/marvell/comphy/phy-comphy-common.h]: Done

File: include/drivers/marvell/ap807_clocks_init.h (hunk 2 above)
Comment (kostapr) [reply include/drivers/marvell/ap807_clocks_init.h]: Done

File: drivers/marvell/ap807_clocks_init.c [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+ */
+
+#include <a8k_plat_def.h>
+#include <aro.h>
+#include <delay_timer.h>
+#include <mmio.h>
+
+/* Notify bootloader on DRAM setup */
+#define AP807_CPU_ARO_CTRL(cluster)	\
+			(MVEBU_RFU_BASE + 0x82A8 + (0xA58 * cluster))
```
Comment (antonio-nino-diaz-arm) [inline drivers/marvell/ap807_clocks_init.c]: ``cluster`` should be enclosed in parentheses.

**Comment by antonio-nino-diaz-arm** [comment]
It looks good to me.

File: drivers/marvell/ap807_clocks_init.c (hunk 4 above)
Comment (kostapr) [reply drivers/marvell/ap807_clocks_init.c]: Done

**Comment by antonio-nino-diaz-arm** [comment]
jenkins: test this please

**Pull request merged**